### DISCUSSION TITLE
广东移动-组播，抓包的源，希望可以添加上适配，造福大家，让广东组播有搜索到，毕竟广东移动的组播还是多人放出来

### DISCUSSION THREAD

**Discussion by lvaneasylam** [Guovin/iptv-api#773] [Q&A] [answered]
肇庆生活服务高清,rtp://239.11.0.54:1025
CGTN俄语高清,rtp://239.11.0.108:1025
CCTV央视台球高清,rtp://239.11.0.60:1025
CCTV-17农村农业高清,rtp://239.10.0.244:1025
CCTV-11戏曲高清,rtp://239.11.0.24:1025
CCTV央视精品高清,rtp://239.21.0.77:3648
江门综合高清,rtp://239.10.0.219:1025
潮州综合高清,rtp://239.21.0.145:3924
河源新闻综合高清,rtp://239.21.0.48:3532
广东移动高清,rtp://239.20.0.133:2064
CGTN西语高清,rtp://239.11.0.105:1025
佛山南海高清,rtp://239.11.0.115:1025
浙江卫视高清,rtp://239.10.0.179:1025
云浮综合高清,rtp://239.11.0.96:1025
CCTV-6电影高清,rtp://239.10.0.103:1025
天津卫视高清,rtp://239.10.0.181:1025
珠海新闻综合高清,rtp://239.10.0.242:1025
四川卫视高清,rtp://239.10.0.204:1025
重庆卫视高清,rtp://239.10.0.205:1025
茂名综合高清,rtp://239.10.0.238:1025
潮州民生高清,rtp://239.21.0.146:3928
肇庆新闻综合高清,rtp://239.11.0.53:1025
广东影视高清,rtp://239.11.0.79:1025
CCTV-2财经高清,rtp://239.10.0.207:1025
江门侨乡生活高清,rtp://239.10.0.139:1025
CCTV-4中文国际高清,rtp://239.10.0.245:1025
广东珠江高清,rtp://239.10.0.62:1025
CCTV卫生健康高清,rtp://239.11.0.185:1025
CCTV-4美洲高清,rtp://239.11.0.102:1025
佛山影视高清,rtp://239.11.0.117:1025
汕尾文化生活高清,rtp://239.21.0.90:3700
云浮文旅高清,rtp://239.11.0.97:1025
茶频道高清,rtp://239.21.0.120:3824
惠州公共高清,rtp://239.21.0.28:3452
广州影视高清,rtp://239.11.0.140:1025
汕头综合高清,rtp://239.21.0.101:3744
深圳卫视高清,rtp://239.10.0.132:1025
广东公共高清,rtp://239.11.0.80:1025
黑龙江卫视高清,rtp://239.10.0.133:1025
中山综合高清,rtp://239.11.0.122:1025
CCTV-9纪录高清,rtp://239.10.0.209:1025
广州南国都市4K,rtp://239.21.0.132:3872
辽宁卫视高清,rtp://239.10.0.185:1025
北京卫视高清,rtp://239.10.0.131:1025
东莞新闻综合高清,rtp://239.21.0.137:3892
CCTV-8电视剧高清,rtp://239.10.0.104:1025
CCTV央视网球高清,rtp://239.11.0.61:1025
云南卫视高清,rtp://239.11.0.171:1025
惠州综合高清,rtp://239.21.0.27:3448
CCTV-12社会与法高清,rtp://239.10.0.211:1025
CCTV第一剧场高清,rtp://239.11.0.67:1025
湖南卫视高清,rtp://239.10.0.115:1025
揭阳生活高清,rtp://239.21.0.92:3708
广东体育高清,rtp://239.20.0.103:2004
广东新闻高清,rtp://239.10.0.199:1025
广东珠江高清,rtp://239.10.0.63:1025
教育卫视高清,rtp://239.10.0.223:1025
嘉佳卡通高清,rtp://239.21.0.94:3716
汕头经济生活高清,rtp://239.21.0.102:3748
CCTV-14少儿高清,rtp://239.10.0.212:1025
东方卫视高清,rtp://239.10.0.183:1025
佛山顺德高清,rtp://239.11.0.116:1025
河源公共高清,rtp://239.21.0.49:3536
求索记录高清,rtp://239.11.0.128:1025
安徽卫视高清,rtp://239.10.0.203:1025
梅州客家生活高清,rtp://239.21.0.150:3944
广东经济科教高清,rtp://239.10.0.112:1025
湖北卫视高清,rtp://239.10.0.134:1025
广州综合高清,rtp://239.11.0.138:1025
江苏卫视高清,rtp://239.10.0.180:1025
江西卫视高清,rtp://239.10.0.80:1025
中国交通高清,rtp://239.10.0.72:1025
东莞生活资讯高清,rtp://239.21.0.138:3896
山东卫视高清,rtp://239.10.0.182:1025
海南卫视高清,rtp://239.11.0.172:1025
广东少儿高清,rtp://239.11.0.81:1025
揭阳综合高清,rtp://239.21.0.91:3704
CCTV-5体育高清,rtp://239.10.0.102:1025
广东卫视高清,rtp://239.10.0.111:1025
CCTV-1综合高清,rtp://239.10.0.114:1025
CCTV电视指南高清,rtp://239.11.0.58:1025
广东经济科教高清,rtp://239.10.0.46:1025
CCTV风云足球高清,rtp://239.11.0.59:1025
CCTV怀旧剧场高清,rtp://239.11.0.64:1025
CCTV风云音乐高清,rtp://239.11.0.68:1025
CCTV-15音乐高清,rtp://239.11.0.25:1025
湖南卫视高清,rtp://239.10.0.49:1025
CCTV兵器科技高清,rtp://239.11.0.57:1025
吉林卫视高清,rtp://239.10.0.81:1025
河北卫视高清,rtp://239.10.0.79:1025
茂名文化生活高清,rtp://239.11.0.99:1025
纪实人文高清,rtp://239.10.0.78:1025
佛山综合高清,rtp://239.11.0.118:1025
东南卫视高清,rtp://239.10.0.77:1025
中山香山文化高清,rtp://239.11.0.123:1025
CCTV女性时尚高清,rtp://239.11.0.62:1025
汕头文旅体育高清,rtp://239.21.0.103:3752
湛江公共高清,rtp://239.10.0.254:1025
CGTN西语高清,rtp://239.21.0.111:3800
清远综合高清,rtp://239.11.0.74:1025
CCTV世界地理高清,rtp://239.11.0.63:1025
广州新闻高清,rtp://239.11.0.139:1025
CGTN纪录高清,rtp://239.21.0.110:3780
CCTV文化精品高清,rtp://239.11.0.66:1025
天津卫视高清,rtp://239.10.0.55:1025
深圳卫视高清,rtp://239.10.0.52:1025
贵州卫视高清,rtp://239.10.0.222:1025
快乐垂钓高清,rtp://239.11.0.131:1025
早期教育高清,rtp://239.11.0.100:1025
CHC影迷电影高清,rtp://239.21.0.119:3820
广州法治高清,rtp://239.11.0.141:1025
大湾区卫视高清,rtp://239.11.0.78:1025
CGTN阿语高清,rtp://239.11.0.107:1025
CCTV风云剧场高清,rtp://239.11.0.65:1025
IPTV5+高清,rtp://239.10.0.218:1025
黑龙江卫视高清,rtp://239.10.0.53:1025
清远文旅生活高清,rtp://239.11.0.75:1025
CCTV-16奥林匹克高清,rtp://239.11.0.126:1025
湖北卫视高清,rtp://239.10.0.54:1025
韶关综合高清,rtp://239.11.0.84:1025
金鹰纪实高清,rtp://239.10.0.206:1025
湛江新闻综合高清,rtp://239.10.0.253:1025
CCTV-10科教高清,rtp://239.10.0.210:1025
CGTN纪录高清,rtp://239.11.0.104:1025
四川卫视高清,rtp://239.10.0.60:1025
梅州综合高清,rtp://239.21.0.149:3940
CCTV-4欧洲高清,rtp://239.11.0.101:1025
金鹰卡通高清,rtp://239.11.0.125:1025
广东体育高清,rtp://239.10.0.47:1025
汕尾新闻综合高清,rtp://239.21.0.89:3696
江苏卫视高清,rtp://239.10.0.51:1025
广西卫视高清,rtp://239.10.0.246:1025
佛山公共高清,rtp://239.11.0.114:1025
CGTN英语高清,rtp://239.11.0.103:1025
河南卫视高清,rtp://239.11.0.94:1025
CCTV-3综艺高清,rtp://239.10.0.101:1025
CCTV-1综合高清,rtp://239.20.0.104:2006
广东卫视高清,rtp://239.20.0.101:2000
广东珠江高清,rtp://239.20.0.64:3144
广东新闻高清,rtp://239.21.0.93:3712
广东民生高清,rtp://239.21.0.86:3684
广东经济科教高清,rtp://239.20.0.102:2002
大湾区卫视高清,rtp://239.21.0.84:3676
广东影视高清,rtp://239.21.0.85:3680
广东少儿高清,rtp://239.21.0.87:3688
岭南戏曲高清,rtp://239.21.0.147:3932
现代教育高清,rtp://239.21.0.148:3936
CCTV-2财经高清,rtp://239.20.0.197:2192
CCTV-3综艺高清,rtp://239.20.0.28:3292
CCTV-4中文国际高清,rtp://239.20.0.16:3340
CCTV-5体育高清,rtp://239.20.0.30:3284
CCTV-6电影高清,rtp://239.20.0.29:3288
CCTV-7军事农业高清,rtp://239.20.0.198:2194
CCTV-8电视剧高清,rtp://239.20.0.31:3280
CCTV-9纪录高清,rtp://239.20.0.199:2196
CCTV-10科教高清,rtp://239.20.0.200:2198
CCTV-11戏曲高清,rtp://239.21.0.35:3480
CCTV-12社会与法高清,rtp://239.20.0.201:2200
CCTV-13新闻高清,rtp://239.21.0.88:3692
CCTV-14少儿高清,rtp://239.20.0.202:2202
CCTV-15音乐高清,rtp://239.21.0.36:3484
CCTV-17农村农业高清,rtp://239.20.0.7:3376
湖南卫视高清,rtp://239.20.0.105:2008
浙江卫视高清,rtp://239.20.0.169:2136
江苏卫视高清,rtp://239.20.0.170:2138
东方卫视高清,rtp://239.20.0.173:2144
安徽卫视高清,rtp://239.20.0.193:2184
北京卫视高清,rtp://239.20.0.121:2040
江西卫视高清,rtp://239.20.0.55:3180
深圳卫视高清,rtp://239.20.0.122:2042
河南卫视高清,rtp://239.21.0.104:3756
云南卫视高清,rtp://239.21.0.133:3876
青海卫视高清,rtp://239.21.0.144:3920
海南卫视高清,rtp://239.21.0.134:3880
甘肃卫视高清,rtp://239.21.0.141:3908
金鹰卡通高清,rtp://239.21.0.115:3804
三沙卫视高清,rtp://239.21.0.143:3916
CETV-4高清,rtp://239.21.0.142:3912
CCTV-5+高清,rtp://239.20.0.203:2204
IPTV5+高清,rtp://239.20.0.208:2214
CCTV-16奥林匹克高清,rtp://239.21.0.116:3808
CCTV-4欧洲高清,rtp://239.21.0.107:3768
CCTV-4美洲高清,rtp://239.21.0.108:3772
CGTN英语高清,rtp://239.21.0.109:3776
CGTN法语高清,rtp://239.21.0.112:3788
CGTN阿语高清,rtp://239.21.0.113:3792
CGTN俄语高清,rtp://239.21.0.114:3796
黑龙江卫视高清,rtp://239.20.0.123:2044
湖北卫视高清,rtp://239.20.0.124:2046
天津卫视高清,rtp://239.20.0.171:2140
山东卫视高清,rtp://239.20.0.172:2142
辽宁卫视高清,rtp://239.20.0.175:2148
四川卫视高清,rtp://239.20.0.194:2186
金鹰纪实高清,rtp://239.20.0.196:2190
重庆卫视高清,rtp://239.20.0.195:2188
贵州卫视高清,rtp://239.20.0.212:2222
CETV-1高清,rtp://239.20.0.213:2224
中国交通高清,rtp://239.20.0.68:3128
东南卫视高清,rtp://239.20.0.58:3168
纪实人文高清,rtp://239.20.0.57:3172
河北卫视高清,rtp://239.20.0.56:3176
吉林卫视高清,rtp://239.20.0.54:3184
广西卫视高清,rtp://239.20.0.15:3344
CCTV兵器科技高清,rtp://239.21.0.80:3660
CCTV电视指南高清,rtp://239.21.0.81:3664
CCTV风云足球高清,rtp://239.21.0.70:3620
CCTV央视台球高清,rtp://239.21.0.71:3624
CCTV女性时尚高清,rtp://239.21.0.73:3632
CCTV世界地理高清,rtp://239.21.0.74:3636
CCTV怀旧剧场高清,rtp://239.21.0.75:3640
CCTV风云剧场高清,rtp://239.21.0.76:3644
CCTV第一剧场高清,rtp://239.21.0.78:3652
CCTV风云音乐高清,rtp://239.21.0.79:3656
早期教育高清,rtp://239.21.0.106:3764
广东体育高清,rtp://239.10.0.113:1025
广东嘉佳卡通高清,rtp://239.10.0.200:1025
岭南戏曲高清,rtp://239.11.0.180:1025
现代教育高清,rtp://239.11.0.181:1025
青海卫视高清,rtp://239.11.0.179:1025
甘肃卫视高清,rtp://239.11.0.175:1025
三沙卫视高清,rtp://239.11.0.177:1025
CETV-4高清,rtp://239.11.0.176:1025
CGTN法语高清,rtp://239.11.0.106:1025
CCTV-1综合高清,rtp://239.20.0.96:3016
广东卫视,rtp://239.20.0.131:2060
广东卫视高清,rtp://239.20.0.99:3004
广东珠江,rtp://239.20.0.111:2020
广东珠江高清,rtp://239.20.0.63:3148
广东新闻,rtp://239.20.0.112:2022
广东民生,rtp://239.20.0.113:2024
广东体育,rtp://239.20.0.114:2026
广东体育高清,rtp://239.20.0.97:3012
广东经济科教,rtp://239.20.0.191:2180
广东经济科教高清,rtp://239.20.0.98:3008
大湾区卫视,rtp://239.20.0.125:2048
广东影视,rtp://239.20.0.117:2032
广东少儿,rtp://239.20.0.118:2034
广东嘉佳卡通,rtp://239.20.0.119:2036
岭南戏曲,rtp://239.20.0.115:2028
现代教育,rtp://239.20.0.128:2054
CCTV-1综合,rtp://239.20.0.192:2182
CCTV-2财经,rtp://239.20.0.176:2150
CCTV-3综艺,rtp://239.20.0.27:3296
CCTV-4中文国际,rtp://239.20.0.177:2152
CCTV-5体育,rtp://239.20.0.26:3300
CCTV-6电影,rtp://239.20.0.25:3304
CCTV-7军事农业,rtp://239.20.0.178:2154
CCTV-8电视剧,rtp://239.20.0.24:3308
CCTV-9纪录,rtp://239.20.0.179:2156
CCTV-10科教,rtp://239.20.0.180:2158
CCTV-11戏曲,rtp://239.20.0.181:2160
CCTV-12社会与法,rtp://239.20.0.182:2162
CCTV-13新闻,rtp://239.20.0.183:2164
CCTV-14少儿,rtp://239.20.0.184:2166
CCTV-15音乐,rtp://239.20.0.185:2168
中国国际电视台,rtp://239.20.0.186:2170
CCTV-17农村农业,rtp://239.20.0.8:3372
湖南卫视,rtp://239.20.0.135:2068
湖南卫视高清,rtp://239.20.0.95:3020
浙江卫视,rtp://239.20.0.106:2010
江苏卫视,rtp://239.20.0.107:2012
东方卫视,rtp://239.20.0.108:2014
安徽卫视,rtp://239.20.0.136:2070
北京卫视,rtp://239.20.0.137:2072
江西卫视,rtp://239.20.0.138:2074
深圳卫视,rtp://239.20.0.134:2066
河南卫视,rtp://239.20.0.144:2086
云南卫视,rtp://239.20.0.145:2088
陕西卫视,rtp://239.20.0.151:2100
山西卫视,rtp://239.20.0.152:2102
内蒙古卫视,rtp://239.20.0.153:2104
青海卫视,rtp://239.20.0.154:2106
海南卫视,rtp://239.20.0.155:2108
宁夏卫视,rtp://239.20.0.156:2110
西藏卫视,rtp://239.20.0.157:2112
新疆卫视,rtp://239.20.0.158:2114
甘肃卫视,rtp://239.20.0.159:2116
兵团卫视,rtp://239.20.0.160:2118
卡酷动画,rtp://239.20.0.162:2122
金鹰卡通,rtp://239.20.0.163:2124
炫动卡通,rtp://239.20.0.164:2126
山东教育,rtp://239.20.0.166:2130
财富天下,rtp://239.20.0.167:2132
厦门卫视,rtp://239.20.0.168:2134
三沙卫视,rtp://239.20.0.52:3192
延边卫视,rtp://239.20.0.51:3216
CETV-2,rtp://239.21.0.46:3524
CETV-4,rtp://239.21.0.47:3528
黑龙江卫视高清,rtp://239.20.0.91:3036
湖北卫视高清,rtp://239.20.0.90:3040
天津卫视高清,rtp://239.20.0.89:3044
金鹰纪实,rtp://239.20.0.85:3060
IPTV谍战剧场,rtp://239.20.0.228:2254
IPTV相声小品,rtp://239.20.0.229:2256
IPTV野外,rtp://239.20.0.230:2258
IPTV法治,rtp://239.20.0.231:2260
IPTV6+,rtp://239.20.0.232:2262
经典电影,rtp://239.20.0.233:2264
IPTV8+,rtp://239.20.0.234:2266
热播剧场,rtp://239.20.0.235:2268
IPTV3+,rtp://239.20.0.236:2270
IPTV5+,rtp://239.20.0.237:2272
IPTV少儿动画,rtp://239.20.0.238:2274
IPTV魅力时尚,rtp://239.20.0.239:2276
汽摩,rtp://239.20.0.222:2242
书画,rtp://239.21.0.140:3904
老故事,rtp://239.20.0.227:2252
中学生,rtp://239.21.0.139:3900
环球奇观,rtp://239.20.0.245:2288
国学频道,rtp://239.20.0.247:2292
发现之旅,rtp://239.21.0.151:3948
广东卫视,rtp://239.10.0.141:1025
广东珠江,rtp://239.10.0.121:1025
广东新闻,rtp://239.10.0.122:1025
广东民生,rtp://239.10.0.123:1025
广东体育,rtp://239.10.0.124:1025
广东经济科教,rtp://239.10.0.201:1025
大湾区卫视,rtp://239.10.0.135:1025
广东影视,rtp://239.10.0.127:1025
广东少儿,rtp://239.10.0.128:1025
广东嘉佳卡通,rtp://239.10.0.129:1025
广东移动,rtp://239.10.0.143:1025
岭南戏曲,rtp://239.10.0.125:1025
现代教育,rtp://239.10.0.138:1025
CCTV-1综合,rtp://239.10.0.202:1025
CCTV-2财经,rtp://239.10.0.186:1025
CCTV-3综艺,rtp://239.10.0.107:1025
CCTV-4中文国际,rtp://239.10.0.187:1025
CCTV-5体育,rtp://239.10.0.108:1025
CCTV-6电影,rtp://239.10.0.109:1025
CCTV-7军事农业,rtp://239.10.0.188:1025
CCTV-8电视剧,rtp://239.10.0.110:1025
CCTV-9纪录,rtp://239.10.0.189:1025
CCTV-10科教,rtp://239.10.0.190:1025
CCTV-11戏曲,rtp://239.10.0.191:1025
CCTV-12社会与法,rtp://239.10.0.192:1025
CCTV-13新闻,rtp://239.10.0.193:1025
CCTV-14少儿,rtp://239.10.0.194:1025
CCTV-15音乐,rtp://239.10.0.195:1025
中国国际电视台,rtp://239.10.0.196:1025
CCTV-17农村农业,rtp://239.10.0.243:1025
湖南卫视,rtp://239.10.0.145:1025
浙江卫视,rtp://239.10.0.116:1025
江苏卫视,rtp://239.10.0.117:1025
东方卫视,rtp://239.10.0.118:1025
安徽卫视,rtp://239.10.0.146:1025
北京卫视,rtp://239.10.0.147:1025
江西卫视,rtp://239.10.0.148:1025
深圳卫视,rtp://239.10.0.144:1025
河南卫视,rtp://239.10.0.154:1025
云南卫视,rtp://239.10.0.155:1025
陕西卫视,rtp://239.10.0.161:1025
山西卫视,rtp://239.10.0.162:1025
内蒙古卫视,rtp://239.10.0.163:1025
青海卫视,rtp://239.10.0.164:1025
海南卫视,rtp://239.10.0.165:1025
宁夏卫视,rtp://239.10.0.166:1025
西藏卫视,rtp://239.10.0.167:1025
新疆卫视,rtp://239.10.0.168:1025
甘肃卫视,rtp://239.10.0.169:1025
兵团卫视,rtp://239.10.0.170:1025
卡酷动画,rtp://239.10.0.172:1025
金鹰卡通,rtp://239.10.0.173:1025
炫动卡通,rtp://239.10.0.174:1025
山东教育,rtp://239.10.0.176:1025
财富天下,rtp://239.10.0.177:1025
厦门卫视,rtp://239.10.0.178:1025
三沙卫视,rtp://239.10.0.83:1025
延边卫视,rtp://239.10.0.140:1025
CETV-2,rtp://239.11.0.35:1025
CETV-4,rtp://239.11.0.36:1025
IPTV谍战剧场,rtp://239.10.0.8:1025
IPTV相声小品,rtp://239.10.0.9:1025
IPTV野外,rtp://239.10.0.10:1025
IPTV法治,rtp://239.10.0.11:1025
IPTV6+,rtp://239.10.0.12:1025
经典电影,rtp://239.10.0.13:1025
IPTV8+,rtp://239.10.0.14:1025
热播剧场,rtp://239.10.0.15:1025
IPTV3+,rtp://239.10.0.16:1025
IPTV5+,rtp://239.10.0.17:1025
IPTV少儿动画,rtp://239.10.0.18:1025
IPTV魅力时尚,rtp://239.10.0.19:1025
汽摩,rtp://239.10.0.232:1025
书画,rtp://239.11.0.174:1025
老故事,rtp://239.10.0.33:1025
中学生,rtp://239.11.0.173:1025
环球奇观,rtp://239.10.0.36:1025
国学频道,rtp://239.10.0.38:1025
发现之旅,rtp://239.11.0.182:1025
河源新闻综合,rtp://239.21.0.50:3540
揭阳综合,rtp://239.21.0.82:3668
揭阳生活,rtp://239.21.0.83:3672
佛山南海,rtp://239.21.0.95:3720
汕头综合,rtp://239.21.0.98:3732
汕头经济生活,rtp://239.21.0.99:3736
汕头文旅体育,rtp://239.21.0.100:3740
广东综艺4K,rtp://239.20.0.60:3160
汕尾新闻综合,rtp://239.20.0.23:3312
汕尾文化生活,rtp://239.20.0.22:3316
天津卫视,rtp://239.20.0.110:2018
山东卫视,rtp://239.20.0.109:2016
湖北卫视,rtp://239.20.0.139:2076
辽宁卫视,rtp://239.20.0.140:2078
黑龙江卫视,rtp://239.20.0.141:2080
贵州卫视,rtp://239.20.0.142:2082
四川卫视,rtp://239.20.0.143:2084
重庆卫视,rtp://239.20.0.146:2090
河北卫视,rtp://239.20.0.147:2092
东南卫视,rtp://239.20.0.148:2094
广西卫视,rtp://239.20.0.149:2096
吉林卫视,rtp://239.20.0.150:2098
深圳卫视高清,rtp://239.20.0.92:3032
CCTV-1综合高清,rtp://239.20.0.32:3276
广东4k超高清4K,rtp://239.20.0.61:3156
广东南方购物高清,rtp://239.20.0.116:2030
CCTV高尔夫·网球高清,rtp://239.21.0.72:3628
求索纪录高清,rtp://239.21.0.118:3816
CCTV-4k超高清4K,rtp://239.21.0.123:3836
CCTV-4k超高清4K,rtp://239.21.0.124:3840
东莞新闻综合,rtp://239.21.0.135:3884
东莞生活资讯,rtp://239.21.0.136:3888
广东卫视高清,rtp://239.10.0.45:1025
CCTV-1综合高清,rtp://239.10.0.48:1025
咪咕赛事4K,rtp://239.10.0.68:1025
CCTV-1综合高清,rtp://239.10.0.100:1025
CCTV-13 新闻高清,rtp://239.10.0.198:1025
CCTV-7 国防军事高清,rtp://239.10.0.208:1025
CCTV-5+ 体育赛事高清,rtp://239.10.0.213:1025
茂名文化生活,rtp://239.10.0.236:1025
茂名综合,rtp://239.10.0.239:1025
江门综合,rtp://239.11.0.69:1025
江门侨乡生活,rtp://239.11.0.70:1025
云浮综合,rtp://239.11.0.76:1025
云浮文旅,rtp://239.11.0.77:1025
佛山南海,rtp://239.11.0.91:1025
佛山公共,rtp://239.11.0.109:1025
佛山南海,rtp://239.11.0.110:1025
佛山顺德,rtp://239.11.0.111:1025
佛山影视,rtp://239.11.0.112:1025
佛山综合,rtp://239.11.0.113:1025
中山综合,rtp://239.11.0.119:1025
中山香山文化,rtp://239.11.0.120:1025
CHC影迷电影高清,rtp://239.11.0.129:1025
茶频道高清,rtp://239.11.0.130:1025
广州竞赛高清,rtp://239.11.0.142:1025
CCTV-4k超高清4K,rtp://239.11.0.161:1025
CCTV-4k超高清4K,rtp://239.11.0.162:1025
广州南国都市4K,rtp://239.11.0.170:1025
中央新影-发现之旅高清,rtp://239.11.0.183:1025

**Comment by Guovin** [comment]
感谢分享

**Reply by lvaneasylam** [reply]
补充两个新抓的

**Reply by lvaneasylam** [reply]
阳江-1,rtp://239.11.0.189:1025
阳江-2,rtp://239.11.0.190:1025

**Comment by buzhixihuan2021** [comment]
感觉现在都是IP限制，不在广东我全部打不开

**Reply by lvaneasylam** [reply]
组播哪有什么这些限制，组播都要经过那个宽带，已经等于落到广东了，关键是有人做udp转换的而已。你说的那个是移动那些盒子链接OTT 吧，不是IPTV 这种

**Reply by buzhixihuan2021** [reply]
明白了，谢谢说明。

**Comment by Guovin** [comment, marked as answer]
rtp文件已更新，但局限于广东移动组播源很少，几乎找不到，目前运行结果暂时还无法用到这部分数据。

**Reply by lvaneasylam** [reply]
也是，我刚刚手动查也是，3个都是电信的
